mk/submodule.mk

```diff
@@ -0,0 +1,9 @@
+# libtuv related variables
+LIBTUV_PATH = deps/libtuv
+LIBTUV_INCLUDE_PATH := $(LIBTUV_PATH)/include
+# PIC (Position-Independent-Code) library
+LIBTUV_LIBRARY := $(LIBTUV_PATH)/build/x86_64-linux/release/lib/libtuv.o
```

Comment:
The generation of `x86_64-linux` is a bit confusing since dcurl itself is portable. Both macOS and GNU/Linux are supported, not to mention arm, aarch64, and x86_64 architecture. We should avoid such hardcoded path though.

Reply:
`libtuv` supports different platforms.
I will copy the corresponding part of `Makefile` in `libtuv` to avoid hardcoded path.

Reply:
It has been fixed in the new commit.